pl_examples/hydra_examples/README.md

```diff
@@ -0,0 +1,9 @@
+## Hydra Pytorch Lightning Example
+
+This directory consists of an example of configuring Pytorch Lightning with (Hydra)[https://hydra.cc/]. Hydra is a tool that allows for the easy configuration of complex applications. 
+
+
+### Tensorboard Visualization
+
+Hydra by default changes the running directory of your program when running into outputs/[YEAR]/[TIME]. Hence, all data with a relative path is submitted into this directory. Therefore to visualize all your tensorboard runs one should run the command: `tensorboard --logdir outputs`. This will then allow you to compare your results across runs.   
```

Comment:
Is it not 
```suggestion
Hydra by default changes the running directory of your program when running into outputs/[DATE]/[TIME]. Hence, all data with a relative path is submitted into this directory. Therefore to visualize all your tensorboard runs one should run the command: `tensorboard --logdir outputs`. This will then allow you to compare your results across runs.   
```

Also, try multirun too.
It adds another level.
```
$ python examples/tutorials/basic/your_first_hydra_app/5_defaults/my_app.py  db=mysql,postgresql -m
[2020-06-23 17:13:33,470][HYDRA] Launching 2 jobs locally
[2020-06-23 17:13:33,470][HYDRA]        #0 : db=mysql
db:
  driver: mysql
  user: omry
  pass: secret

[2020-06-23 17:13:33,568][HYDRA]        #1 : db=postgresql
db:
  driver: postgresql
  user: postgre_user
  pass: drowssap
  timeout: 10

omry@Coronadev:~/dev/hydra(init_api_changes)$ tree multirun/
multirun/
└── 2020-06-23
    └── 17-13-33
        ├── 0
        │   └── my_app.log
        ├── 1
        │   └── my_app.log
        └── multirun.yaml

```

Reply:
This is really cool, I never even knew you could do this with Hydra! I tried it out and yes, multirun works with tensorboard.
`tensorboard --logdir multirun/`